conf/modules.config

```diff
@@ -465,6 +465,34 @@ process {
         ]
     }
 
+    withName: ANGSD_DOCOUNTS {
+        ext.args = [
+            "-iCounts 1",
+            "-r ${params.angsd_chrom_name}:5000000-154900000",
```

Comment:
Could you also add the range as a parameter, where the defaults is `5000000-154900000`?
Will be a bit of a headache with multi-reference mapping, but it is the least restrictive way to implement this. Users might want to use nuclear contamination on birds or whatever